### PR TITLE
Add vagrant example configuration variables.

### PR DESCRIPTION
**Pull request by jlongever**
- Add bridge interface variable
- Add workspace and target repo variable

@RackHD/corecommitters @DavidjohnBlodgett @keedya @zyoung51 @uppalk1 @tannoa2

### CODE REVIEW

File: example/Vagrantfile [hunk 1]
```diff
@@ -15,12 +15,34 @@ Vagrant.configure(VAGRANTFILE_API_VERSION) do |config|
             v.customize ["modifyvm", :id, "--nicpromisc2", "allow-all"]
         end
 
+        # Sync optional local workspace directory to the vagrant instances src/ path.
+        # REPO is optional when WORKSPACE is defined, used for synchronizing single RackHD repos 
+        # otherwise it assumes all service repos have been cloned and built under the WORKSPACE path.
+        if ENV['WORKSPACE']
+	  if ENV['REPO'] == "on-tasks"
+            target.vm.synced_folder "#{ENV['WORKSPACE']}/#{ENV['REPO']}", "/home/vagrant/src/on-taskgraph/node_modules/on-tasks/"
```
Comment (benbp) [inline example/Vagrantfile]: on-tasks should be linked to on-http as well

Comment (jlongever) [inline example/Vagrantfile]: thank you sir.

**Comment by jlongever** [comment]
@benbp updated.

**Comment by benbp** [comment]
Thanks! :+1:

The optional bridging set up option is great. I bet it means I can change the config and do vagrant provision instead of having to muck with the GUI or vboxmanage tool when I want to change the network setup of my VM.

**Comment by zyoung51** [comment]
+1

**Comment by jlongever** [comment]
@heckj does this look good to you?

**Comment by heckj** [comment]
:+1: nicely set up!

**Pull request merged**